src/pepr/operator/crd/generated/exemption-v1alpha1.ts

```diff
@@ -0,0 +1,83 @@
+// This file is auto-generated by kubernetes-fluent-client, do not edit manually
+
+import { GenericKind, RegisterKind } from "kubernetes-fluent-client";
+
+export class Exemption extends GenericKind {
+  spec?: Spec;
+  status?: Status;
+}
+
+export interface Spec {
+  /**
+   * Policy exemptions
+   */
+  exemptions?: ExemptionElement[];
+}
+
+export interface ExemptionElement {
+  /**
+   * Reasons as to why this exemption is needed
+   */
+  description?: string;
+  /**
+   * Resource to exempt (Regex allowed for name)
+   */
+  matcher: Matcher;
+  /**
+   * A list of policies to override
+   */
+  policies: Policy[];
+  /**
+   * title to give the exemption for reporting purposes
+   */
+  title?: string;
+}
+
+/**
+ * Resource to exempt (Regex allowed for name)
+ */
+export interface Matcher {
+  kind?: Kind;
+  name: string;
+  namespace: string;
+}
+
+export enum Kind {
+  Pod = "pod",
+  Services = "services",
+}
+
+export enum Policy {
+  DisallowHostNamespaces = "Disallow_Host_Namespaces",
+  DisallowNodePortServices = "Disallow_NodePort_Services",
+  DisallowPrivileged = "Disallow_Privileged",
+  DisallowSELinuxOptions = "Disallow_SELinux_Options",
+  DropAllCapabilities = "Drop_All_Capabilities",
+  RequireNonRootUser = "Require_Non_Root_User",
+  RestrictCapabilities = "Restrict_Capabilities",
+  RestrictExternalNames = "Restrict_External_Names",
+  RestrictHostPathWrite = "Restrict_HostPath_Write",
+  RestrictHostPorts = "Restrict_Host_Ports",
+  RestrictProcMount = "Restrict_Proc_Mount",
+  RestrictSELinuxType = "Restrict_SELinux_Type",
+  RestrictSeccomp = "Restrict_Seccomp",
+  RestrictVolumeTypes = "Restrict_Volume_Types",
+}
```

Comment:
Definitely not the "convention" expert here, but this feels a bit strange to use `PascalCase` for the enum names but `_` for the value? Don't want to keep asking about naming conventions 🙃, but the remote generated stuff for example uses pascal for both. Tracking that this would need updates across a number of files if we change it though...